tests/e2e/e2e_workflow.spec.js

```diff
@@ -62,7 +62,7 @@ describe('Basic end-to-end Workflow', function () {
 
     // Can't user the `ng` helper because somewhere the environment gets
     // stuck to the first build done
-    sh.exec('./node_modules/.bin/ng build --environment=production --silent');
+    sh.exec(`node_modules${path.sep}.bin${path.sep}ng build --environment=production --silent`);
```

Comment:
Yes, that is the reason. Now that you mention it, since the e2e tests run npm link, a simple `ng build` should work as well.

I'll make another PR to simply this.